mlptrain/sampling/md.py

```diff
@@ -24,7 +26,8 @@ def run_mlp_md(configuration: 'mlptrain.Configuration',
                init_temp:     Optional[float] = None,
                fbond_energy:  Optional[dict] = None,
                bbond_energy:  Optional[dict] = None,
-               bias:          Optional['mlptrain.bias.Bias'] = None,
+               bias:          Optional[Union['mlptrain.Bias',
+                                             'mlptrain.PlumedBias']] = None,
                method:        Optional[str] = None,
```

Comment:
I would only allow valid options. At the moment people can type anything in and it won't immediately break if it isn't implemented, which might lead to confusion over what is breaking. Also for strings, lower() so people can type in any case